landscapes.yml

```diff
@@ -19,6 +19,10 @@ landscapes:
     repo: cdfoundation/cdf-landscape
     hook: 5d8e8ecbeb12d77369f06dba
     required: true
+  - landscape:
+    name: finos-landscape
+    repo: finos/FINOS-landscape
```

Comment:
@caniszczyk following up on the conversation [here](https://github.com/finos/finos-landscape/issues/31#issuecomment-651286007), from this yaml, it seems like we could have

name: finos-landscape
repo: finos/landscape

Would that work or still break stuff?